recipes/jwt-cpp/all/patches/0003-fix-picojson-header-location-for-conan.patch

```diff
@@ -0,0 +1,6 @@
+diff --git a/include/jwt-cpp/jwt.h b/include/jwt-cpp/jwt.h
+--- a/include/jwt-cpp/jwt.h
++++ b/include/jwt-cpp/jwt.h
+@@ -3,1 +3,1 @@
+-#include "picojson/picojson.h"
++#include "picojson.h"
```

Comment:
Did you create this patch or is it from the upstream? Asking because maybe we could fix it in picojson, by cpp_info.includedirs, so we don't need this patch.

Reply:
I created =D picojson leaves this completely undefined.

reading the [docs](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info) I would not have expected it to add directory paths.